### ISSUE TITLE
Create PR to add automatic image builds for tagged "versions" of SlideTap

### ISSUE THREAD

**Issue by pontus** [NBISweden/aida-data-hub#309]
For Slides, it's probably good to have automatic image builds, at least for tagged commits. Would be nice to have automatic (through github actions).

**Comment by yohell** [comment]
This is in the sprint backlog, but is labeled Not yet refined. Should we discuss in refinement?

**Comment by pontus** [comment]
Wait on opening the product for now.
